scene/gui/split_container.cpp

```diff
@@ -359,6 +359,10 @@ bool SplitContainer::is_collapsed() const {
 	return collapsed;
 }
 
+bool SplitContainer::is_autohide() const {
```

Comment:
The new method is not needed. You can access private fields of objects of the same class.

Reply:
I wasn't sure how it would work because it's a different instance of the same class. Is this still allowed?

Reply:
Yes.